ready for review

Michael-Ivlev opened the pull request.

Michael-Ivlev committed:
ready for review

Michael-Ivlev committed:
last fix before review

change the fonts directory

ImaliaNsk commented:
Great job! A few issues will need correcting, but on the whole, the project looks very impressive! Below is the checklist with requirements and comments, I hope the issues marked will be easy to fix!

# Project review summary:
### Performance Criteria
- [x] <!--14.429--> The project is adapted for different screen resolutions and matches the designs for these resolutions. There is no horizontal scrolling on resolutions of 320px and up. The scrollbar can't be hidden using the overflow: hidden declaration.

* There should be a link to open "Sign in" popup http://joxi.ru/4AkWX4XijM6vNA
* Side margin in missing http://joxi.ru/KAgwR4RC5gvpOA. The same screenshot shows that on some resolutions the images inside cards have different height. Please fix, all cards should have the same layout, the images should have similar size.

- [x] <!--14.429--> All the blocks from the brief have been coded. Navigation between pages and links to external resources are working correctly: there are no broken links or links leading to an anchor on the page and external links open in a new tab.

* footer should contain working links opening in new tabs.

- [x] <!--14.429--> The layout is responsive and displays correctly at all intermediate resolutions.
- [x] <!--3.38--> There are no validation errors.
- [x] <!--3.38--> Semantic HTML is used, meaning that semantic tags are used. All elements are used correctly (e.g. a paragraph must be a paragraph, a list must be a list). The DOM tree structure doesn't consist only of <div> containers.

* Navigation menu in header and footer should be nav, or a list, or a list inside nav.
* The page should contain only one h1
* Markup in the newscardlist needs changing: please either use `article` for the news cards, or make the list with `ul` instead of `div` (you can also do both)
* footer should be a `footer`

- [x] <!--3.38--> The correct approach is chosen for positioning elements, and is described using the correct syntax. For example, when positioning an element absolutely, its parent block is relatively positioned.
- [x] <!--3.38--> flex or grid layout are used to arrange elements.
- [x] <!--3.38--> Infrastructural project files are created using CRA.
- [x] <!--3.38--> The popup windows have opening and closing functionality implemented.
- The markup has been converted into JSX:
	- [x] <!--3.38--> The markup is included within parentheses ( ).
	- [x] <!--3.38--> The markup has been moved into the corresponding components.	
- Your project contains:
	- [x] <!--3.38--> An images folder
	- [x] <!--3.38--> A components folder with JS and CSS files for the components.
	- [x] <!--3.38--> A fonts folder.

### Best Practices
- [ ] <!--2.143--> Classes are named according to BEM specifications.

* http://joxi.ru/xAe4ZqZcMYjo42 Elements can't be used outside of their blocks. It's also necessary to use a boolean modifier instead of another element for the active state of the link.
* incorrectly used modifiers http://joxi.ru/a2XwjRjCQybRNm http://joxi.ru/nAyyl4lu9X1xYA http://joxi.ru/ZrJwxlxCQ1pVbm http://joxi.ru/BA0xqPqH1BWyYA.

- [x] <!--2.143--> Buttons, input fields, and links are implemented in all of the states specified in the design.
- [x] <!--2.143--> The design reuses components wherever possible.
- [x] <!--2.143--> Fonts are connected using @font-face.
- [x] <!--2.143--> The use of reset.css is prohibited.
- [x] <!--2.143--> Form elements should be highlighted when focused.
- [x] <!--2.143--> The form must have placeholders, and required fields should be validated.

### Recommendations
- [x] <!--1.67--> Images have an alt attribute containing appropriate values.

* Image in `about`, `nothing-found` sections

- [x] <!--1.67--> System fonts are connected as alternatives to each of your fonts.

* Please, correct all similar cases http://joxi.ru/82QwJ6JC81OQlr

- [x] <!--1.67--> Raster and vector images have been optimized.

## Number of points: 63

Michael-Ivlev committed:
after review

- fix margin
- fix div tags to nav and footer
- ass alt to images
- add alternative fonts

ImaliaNsk commented:
Congratulations, the project is accepted! I wish you good luck on the next stage!

# Project review summary:
### Performance Criteria
- [x] <!--14.429--> The project is adapted for different screen resolutions and matches the designs for these resolutions. There is no horizontal scrolling on resolutions of 320px and up. The scrollbar can't be hidden using the overflow: hidden declaration.
- [x] <!--14.429--> All the blocks from the brief have been coded. Navigation between pages and links to external resources are working correctly: there are no broken links or links leading to an anchor on the page and external links open in a new tab.

* footer should contain working links opening in new tabs (link "Practicum by Yandex" doesn't work).

- [x] <!--14.429--> The layout is responsive and displays correctly at all intermediate resolutions.
- [x] <!--3.38--> There are no validation errors.
- [x] <!--3.38--> Semantic HTML is used, meaning that semantic tags are used. All elements are used correctly (e.g. a paragraph must be a paragraph, a list must be a list). The DOM tree structure doesn't consist only of <div> containers.
- [x] <!--3.38--> The correct approach is chosen for positioning elements, and is described using the correct syntax. For example, when positioning an element absolutely, its parent block is relatively positioned.
- [x] <!--3.38--> flex or grid layout are used to arrange elements.
- [x] <!--3.38--> Infrastructural project files are created using CRA.
- [x] <!--3.38--> The popup windows have opening and closing functionality implemented.
- The markup has been converted into JSX:
	- [x] <!--3.38--> The markup is included within parentheses ( ).
	- [x] <!--3.38--> The markup has been moved into the corresponding components.	
- Your project contains:
	- [x] <!--3.38--> An images folder
	- [x] <!--3.38--> A components folder with JS and CSS files for the components.
	- [x] <!--3.38--> A fonts folder.

### Best Practices
- [ ] <!--2.143--> Classes are named according to BEM specifications.

* http://joxi.ru/xAe4ZqZcMYjo42 Elements can't be used outside of their blocks. It's also necessary to use a boolean modifier instead of another element for the active state of the link.
* incorrectly used modifiers http://joxi.ru/a2XwjRjCQybRNm http://joxi.ru/nAyyl4lu9X1xYA http://joxi.ru/ZrJwxlxCQ1pVbm http://joxi.ru/BA0xqPqH1BWyYA.

- [x] <!--2.143--> Buttons, input fields, and links are implemented in all of the states specified in the design.
- [x] <!--2.143--> The design reuses components wherever possible.
- [x] <!--2.143--> Fonts are connected using @font-face.
- [x] <!--2.143--> The use of reset.css is prohibited.
- [x] <!--2.143--> Form elements should be highlighted when focused.
- [x] <!--2.143--> The form must have placeholders, and required fields should be validated.

### Recommendations
- [x] <!--1.67--> Images have an alt attribute containing appropriate values.
- [ ] <!--1.67--> System fonts are connected as alternatives to each of your fonts.

* Please, correct all similar cases http://joxi.ru/YmEwlxlCGZdvPA

- [x] <!--1.67--> Raster and vector images have been optimized.

## Number of points: 82

Michael-Ivlev committed:
final fix

- add alternative system fonts
